core/Helper/Data.php

```diff
@@ -350,12 +352,37 @@ public function getFormKey()
         return $this->formKey->getFormKey();
     }
 
+    /**
+     * Get the default first name for all dealers
+     * @return string
+     */
+    public function getDefaultFirstName()
+    {
+        $quote = $this->checkoutSession->getQuote();
+        $customer = $quote->getCustomer();
+
+        if ($customer && $customer->getId()) {
+            $firstName = $customer->getFirstname();
```

Comment:
You can `return $customer->getFirstname();` instead of assigning the value to $firstName